src/pages/study-room/components/timer.tsx

```diff
@@ -1,15 +1,47 @@
 import { Button, Card, CardActions, CardContent, CardHeader } from '@mui/material';
+import { useMemo } from 'react';
+import useTimer from '../hook/useTimer';
+
+type KeyType = { [key in string]: string };
+const STEP_TEXT: KeyType = {
+  PLANNING: '계획',
+  STUDYING: '스터디',
+  RETROSPECT: '회고',
+  RESTING: '휴식',
+  COMPLETED: '완료',
+};
```

Comment:
constant에 step이 있어서 요것도 그쪽으로 모아두면 어떨까 싶네요~

Reply:
이렇게 표현해줘도 좋을 것 같아요~
```suggestion
const STEP_TEXT: Record<string, string> = {
  PLANNING: '계획',
  STUDYING: '스터디',
  RETROSPECT: '회고',
  RESTING: '휴식',
  COMPLETED: '완료',
};
```

Reply:
STEP_TEXT의 key값은 타입을 잡아줄 수 있을 것 같네요!

```ts
type StepTextKey = keyof typeof STEP;

const STEP_TEXT: Record<StepTextKey, string> = {
  PLANNING: '계획',
  STUDYING: '스터디',
  RETROSPECT: '회고',
  RESTING: '휴식',
  COMPLETED: '완료',
};
```

Reply:
보통 요런 객체는 MAP이라는 이름으로도 많이 표현해주는 것 같습니다~
```suggestion
const STEP_LABEL_MAP: KeyType = {
  PLANNING: '계획',
  STUDYING: '스터디',
  RETROSPECT: '회고',
  RESTING: '휴식',
  COMPLETED: '완료',
};
```